internal/cli/atlas/clusters/create.go

```diff
@@ -64,6 +64,8 @@ func (opts *CreateOpts) initStore() error {
 	return err
 }
 
+var createTmpl = "Deploying cluster {{.Name}}.\n"
```

Comment:
[q] Thinking aloud - I wonder should all these templates defined in one place and pulled in, rather than in their particular module.

My reasoning is consistency of messaging would be easier to maintain in this manner as you are looking at all templates at the same time.

It's probably not a very go like approach though - what do you think?

Reply:
you're right there in that's not very go like, we have a global "[flag](https://github.com/mongodb/mongocli/tree/master/internal/flag)" package for the same reason you describe and I was temped to follow the same approach but that `flag` and `usage` packages are becoming unmaintainable and a dumping ground of `const`s and I've been thinking if it would be worth to start splitting them

This is my long way of saying, I see your point but I don't know which one is best

Reply:
(side note as I open the editor) the advantage of having the template in the command is that you have easier access to see the fields of the struct you're templating) so I think I vote on keeping it here

Reply:
It's all trade-offs :) To be clear, I have no problem with the current approach, just throwing ideas around. The fact that you've already tried with flags and are reconsidering the approach is evidence enough for me to keep going this the approach in this PR.